server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -1162,6 +1174,10 @@ public IndexWriteLoad getWriteLoad() {
         return writeLoad;
     }
 
+    public OptionalDouble getForecastedWriteLoad() {
```

Comment:
I think it would be good to mark this as a forbidden API so we don't inadvertently call it directly, which would bypass the license check built into `WriteLoadForecaster#getForecastedWriteLoad`.